src/attribute-editor/interfaces.ts

```diff
@@ -37,7 +37,14 @@ export namespace AttributeEditorProps {
   }
 
   export interface Ref {
+    /**
+     * Focuses the remove button for the given row index.
+     */
     focusRemoveButton(itemIndex: number): void;
+    /**
+     * Focuses the 'add' button. Use this, for example, after a user removes the last row.
```

Comment:
minor/writing: They should probably either both say "remove button" and "add button", or "'remove' button" and "'add' button"